src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -531,145 +520,208 @@ private RecommendedSymbols GetSymbolsOffOfConditionalReceiver(ExpressionSyntax o
             ITypeSymbol? containerType,
             bool unwrapNullable,
-            bool isForDereference)
+            bool isForDereference,
+            bool allowColorColor)
         {
-            var excludeInstance = false;
-            var excludeStatic = true;
-            var excludeBaseMethodsForRefStructs = true;
+            var result = GetSymbolsOffOfBoundExpressionWorker(leftHandBinding);
+            if (!allowColorColor || !CanAccessInstanceAndStaticMembersOffOf(out var reinterpretedBinding))
+                return result;
+
+            var typeMembers = GetSymbolsOffOfBoundExpressionWorker(reinterpretedBinding);
 
-            ISymbol? containerSymbol = containerType;
+            return new RecommendedSymbols(
+                result.NamedSymbols.Concat(typeMembers.NamedSymbols),
+                result.UnnamedSymbols);
 
-            var symbol = leftHandBinding.GetAnySymbol();
-            if (symbol != null)
+            bool CanAccessInstanceAndStaticMembersOffOf(out SymbolInfo reinterpretedBinding)
             {
-                // If the thing on the left is a lambda expression, we shouldn't show anything.
-                if (symbol is IMethodSymbol { MethodKind: MethodKind.AnonymousFunction })
-                    return default;
+                reinterpretedBinding = default;
 
-                var originalExpressionKind = originalExpression.Kind();
+                // Check for the Color Color case.
+                //
+                // color color: if you bind "A" and you get a symbol and the type of that symbol is
+                // Q; and if you bind "A" *again* as a type and you get type Q, then both A.static
+                // and A.instance are permitted
+                if (expression is not IdentifierNameSyntax identifier)
+                    return false;
 
-                // If the thing on the left is a type, namespace or alias and the original
-                // expression was parenthesized, we shouldn't show anything in IntelliSense.
-                if (originalExpressionKind is SyntaxKind.ParenthesizedExpression &&
-                    symbol.Kind is SymbolKind.NamedType or SymbolKind.Namespace or SymbolKind.Alias)
-                {
-                    return default;
-                }
+                var semanticModel = _context.SemanticModel;
+                var symbol = leftHandBinding.GetAnySymbol();
 
-                // If the thing on the left is a method name identifier, we shouldn't show anything.
-                if (symbol.Kind is SymbolKind.Method &&
-                    originalExpressionKind is SyntaxKind.IdentifierName or SyntaxKind.GenericName)
+                // If the symbol is currently bound as a named type, try to bind it as an instance.  Conversely, if it's
+                // bound as an instance, try to bind it as a named type.
+                INamedTypeSymbol? instanceType, staticType;
+                if (symbol is INamedTypeSymbol namedType)
```

Comment:
OK, so the issue is caused by the node being bound as a type, which our previous logic didn't handle?

Reply:
the opposite :).  We bound it as an instance, so we didn't show the static member.  

The prior logic said "if we bound it as a type, also try as an instance".  Now we have the reverse "if we bound it as an instance, also try it as a type".